### ISSUE TITLE
Define the requirements

### ISSUE THREAD

**Issue by CodingIsLove** [CodingIsLove/RadiationNation#5]
### Story description
Before starting to code our game, it is required to make a clean list of our requirements. Define all the requirements for the project and split them up in the three categories:
1. required 
2. nice-to-have
3. optional

### Goal
- [x] In the ReadMe, a link will lead you to the requirements page
- [x] All the requirements (for the three classes) are clearly defined and understandable.
